_sections/merge-conflicts-android-studio.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Android Studio
+tab: android
+---
+
+### Overview
+
+GitLive allows you to see your teammates’ changes to a file in the gutter of your editor and get notified of conflicts as soon as they occur, even across branches and uncommitted changes.
+
+![jetbrains Gutter](/uploads/jetbrains-gutter-cherry-pick.gif "jetbrains Gutter"){:class="screenshot"}
+### Gutter indicators for teammate’s changes
+GitLive adds gutter indicators to show you where your teammate’s have made changes to the file you have open, these update in real-time as you and your teammates are editing.
+
+Insertions are represented by a green arrow, modifications are denoted by a blue line and deletions by a  grey line.
```

Comment:
deletions in red is that right? i thought conflicts are red?

Reply:
i think your right maybe crop of the lower part of the screenshot as the bottom red indicator is a conflict. so that they are not confused